snapcraft/internal/parser.py

```diff
@@ -18,6 +18,8 @@
 """
 snapcraft-parser
 
+Parse a wiki index of parts and output formated YAML for use by 'snapcraft'
```

Comment:
Could you explain how snapcraft uses the formatted YAML in a few words? Or would that be too much info?